doc/user/content/sql/show-clusters.md

```diff
@@ -17,53 +17,55 @@ menu:
 
 ## Pre-installed clusters
 
-Materialize has several pre-installed clusters.
+When you enable a Materialize region, several clusters that are used to improve
+the user experience, as well as support system administration tasks, will be
+pre-installed.
 
-### Default cluster
+### `default` cluster
 
-When you enable a Materialize region, a cluster named `default` with a single
-`xsmall` [replica](/overview/key-concepts/#cluster-replicas) named `r1` will be
-pre-installed. You can modify or drop this cluster or its replicas at any
-time.
+A cluster named `default` with a single `xsmall` [replica](/overview/key-concepts/#cluster-replicas)
+named `r1` will be pre-installed in every environment. You can modify or drop
+this cluster or its replicas at any time.
 
-{{< note >}} The default value for the `cluster` session parameter is `default`.
+{{< note >}}
+The default value for the `cluster` session parameter is `default`.
 If the `default` cluster is dropped, you must run [`SET cluster`](/sql/select/#ad-hoc-queries)
-to choose a valid cluster in order to run `SELECT` queries.{{< /note >}}
+to choose a valid cluster in order to run `SELECT` queries.
+{{< /note >}}
 
 ### `mz_introspection` system cluster
 
-The `mz_introspection` cluster has several indexes installed by default that
-speed up common `SHOW` statements. You are not billed for this cluster.
-It has the following restrictions:
+A system cluster named `mz_introspection` will be pre-installed in every
+environment. This cluster has several indexes installed to speed up common
+introspection queries, like `SHOW` commands and queries using the system
+catalog. It is recommended to switch to the `mz_introspection` cluster for
+improved performance when executing these statements.
 
+The following characteristics apply to the `mz_introspection` cluster:
+
+  * You are **not** billed for this cluster.
   * You cannot create indexes or materialized views on this cluster.
   * You cannot drop this cluster.
-
-You are, however, are permitted to run `SELECT` or `SUBSCRIBE` statements on
-this cluster.
-
-You are encouraged to use this cluster when running queries against the
-system catalog:
-
-```sql
-SET cluster = mz_introspection;
-SELECT * FROM mz_sources;
-SHOW CLUSTERS;
-```
+  * You can run `SELECT` or `SUBSCRIBE` statements on this cluster.
 
 ### `mz_system` system cluster
 
-The `mz_system` cluster is used for various internal system tasks. You are not
-billed for this cluster. You are not permitted to use this cluster.
-Specifically:
+A system cluster named `mz_system` will be pre-installed in every environment.
+This cluster is used for different system administration tasks, like billing
```

Comment:
"different system administration tasks" here raises the question of "different from what?" I'd also prefer to be somewhat circumspect with our description of its purpose; the idea is that it's running who knows what to support the system and you, as the end user, shouldn't know or care. What do you think about saying just "This cluster is used by the system for various internal monitoring tasks", or something like that?

Reply:
Yeah, I went back and forth on it; especially about mentioning billing. @heeringa asked if we could be clearer about the purpose, but...maybe we just don't need to be.

Reply:
LGTM